### DISCUSSION TITLE
Error when adding file +page.server.js

### DISCUSSION THREAD

**Discussion by devappbox** [sveltejs/kit#8682] [Q&A]
I follow this article https://dev.to/aryaprakasa/serving-single-page-application-in-a-single-binary-file-with-go-12ij. And this is the repo https://github.com/aprakasa/go-embed-spa. 

But I got an error when adding file +page.server.js in routes/about folder.

``````
+page.server.js
/** @type {import('./$types').PageServerLoad} */

export async function load() {
return {}
}

``````
Here's the error in the browser console

``````
> Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/html". Strict MIME type checking is enforced for module scripts per HTML spec.

> start-ce5b08e9.js:1 TypeError: Failed to fetch dynamically imported module: http://localhost:5050/about/__data.js?__invalid=ny&__id=1

``````
hope someone to provide me a workaround for this case.

Thank you.

**Comment by teemingc** [comment]
I suspect this issue is caused by how the Go server is serving the SvelteKit build files.

You may have to re-configure your Go server.
